lib/view/pages/game_library/game_library_page.dart

```diff
@@ -143,8 +139,8 @@ class _GameLibraryState extends State<GameLibrary> {
             body: Column(
               children: [
                 TabBar(tabs: const [
-                  Tab(text: 'Games'),
-                  Tab(text: 'Connections'),
+                  Tab(text: 'GAMES'),
```

Comment:
Small thing , could we perhaps leave these as Games and Connections, as this was something our mentors mentioned?

Reply:
I don't know what you mean here.  It's already like that

Reply:
It should be Connections and Games , not CONNECTIONS and GAMES

Reply:
Hmm, in my code its already like that